tensorflow/c/experimental/stream_executor/BUILD

```diff
@@ -59,3 +60,11 @@ tf_cc_test(
         "//tensorflow/stream_executor:stream_executor_pimpl",
     ],
 )
+
+exports_files(
```

Comment:
@annarev  I saw c0904b9 expose the stream_executor_hdrs as follows, can deps[c_api_macros/ tf_status] change to headers[c_api_macro.h/tf_status.h]? since stream_executor.h needs its header not the module. With this, I met an issue when change the test_pluggbale_device.so BUILD file(stream_executor.h->stream_executor_hdrs), it reports a lot of multiple definition error. it was due to tf_status depends on tensorflow/core:lib and seems also build in /cuda/lib/libcudart_static.a. 
```
cc_library(
    name = "stream_executor_hdrs",
    hdrs = ["stream_executor.h"],
    visibility = ["//tensorflow:internal"],
    deps = [
        "//tensorflow/c:c_api_macros",
        "//tensorflow/c:tf_status",
    ],
)
```
error message:
```
  external/local_config_cuda/crosstool/clang/bin/crosstool_wrapper_driver_is_not_gcc @bazel-out/k8-opt/bin/tensorflow/c/experimental/stream_executor/test/test_pluggable_device.so-2.params)
Execution platform: @local_execution_config_platform//:platform
bazel-out/k8-opt/bin/external/local_config_cuda/cuda/cuda/lib/libcudart_static.a(libcudart_static.a.o): In function `cudaGetDevice':
(.text+0x3ce50): multiple definition of `cudaGetDevice'
bazel-out/k8-opt/bin/tensorflow/stream_executor/cuda/libcudart_stub.pic.a(cudart_stub.pic.o):cudart_stub.cc:(.text.cudaGetDevice+0x0): first defined here
bazel-out/k8-opt/bin/external/local_config_cuda/cuda/cuda/lib/libcudart_static.a(libcudart_static.a.o): In function `cudaStreamAddCallback':
(.text+0x436d0): multiple definition of `cudaStreamAddCallback'
bazel-out/k8-opt/bin/tensorflow/stream_executor/cuda/libcudart_stub.pic.a(cudart_stub.pic.o):cudart_stub.cc:(.text.cudaStreamAddCallback+0x0): first defined here
bazel-out/k8-opt/bin/external/local_config_cuda/cuda/cuda/lib/libcudart_static.a(libcudart_static.a.o): In function `cudaMemcpyArrayToArray':
(.text+0x2c080): multiple definition of `cudaMemcpyArrayToArray'
```